MigrationV3V4/CSharp/Skills/V3PizzaBot/PizzaOrderDialog.cs

```diff
@@ -99,7 +99,7 @@ private async Task PizzaFormComplete(IDialogContext context, IAwaitable<PizzaOrd
             }
 
             // When the skill completes, send an `endOfConversation` activity and include the finished order.
-            await SkillsHelper.EndSkillConversation(context.Activity as Activity, order);
+            await ConversationHelper.EndConversation(context.Activity as Activity, order);
```

Comment:
Shouldn't you also call clear conversation state after this?

Reply:
The Form itself has been completed, so this will clear the state.